app/services/org/create_last_month_created_plan_service.rb

```diff
@@ -2,30 +2,60 @@ class Org
   class CreateLastMonthCreatedPlanService
     class << self
       def call(org = nil)
-        orgs = org.nil? ? ::Org.all : [org]
+        orgs = org.nil? ? Org.all : [org]
 
         orgs.each do |org|
           months = OrgDateRangeable.split_months_from_creation(org)
           last = months.last
           if last.present?
-            create_count_for_date(start_date: last[:start_date], end_date: last[:end_date], org: org)
+            create_or_update(start_date: last[:start_date], end_date: last[:end_date], org: org)
           end
         end
       end
 
       private
 
+      def users(org)
+        User.where('users.org_id = ?', org.id)
+      end
+
+      def plans(start_date:, end_date:)
+        Plan.where('plans.created_at >= ? AND plans.created_at <= ?', start_date, end_date)
```

Comment:
See comments in `create_created_plan_service`. I think there is a more DRY solution. This code looks identical.